code/modules/mob/living/simple_animal/friendly/nian_caterpillar.dm

```diff
@@ -83,6 +83,8 @@
 
 	// Mind transfer to new worme.
 	M.transfer_to(adult)
+	// Doesn't allow cult or vampires to benefit from them fully, similar to "TRAIT_HOLY".
```

Comment:
same with this comment

Reply:
Changed the comments to be more specific. Let me know if that's good :)